### ISSUE TITLE
Great package but spurious version numbers

### ISSUE THREAD

**Issue by aubreyz** [lexicalunit/atom-notes#63]
Why on last two versions has the only change been the version number??

Or am I misinterpreting something

Great package otherwise

**Comment by lexicalunit** [comment]
The [last two commits](https://github.com/lexicalunit/atom-notes/commits/master)? Looking back it was 9 days ago, a change to the dependencies initiated by dependabot. I'm not sure where you're looking tho. Do you have a link?

**Comment by aubreyz** [comment]
I'm probably misinterpreting - if I go to https://github.com/lexicalunit/atom-notes/releases
and click on  e658b00 (for example) and look at the diffs - it shows two changed files both of which are just changes to the version number.  Do you have release notes somewhere? - I generally don't like to update stuff unless I know why.

**Comment by lexicalunit** [comment]
Ah, that seems to be the way `apm` does things. What you want to look at is the diff from that release to the previous: https://github.com/lexicalunit/atom-notes/compare/v1.21.0...v1.21.1 &emdash; from this view you can see that the changes are the dependabot dependency update I mentioned plus the separate commit to prepare the release.

You can get to this view from any particular release page, then click the `compare` drop down and select the previous release.

**Comment by lexicalunit** [comment]
Oh wait a minute... I see what you mean:

![Screen Shot 2020-03-26 at 9 29 22 AM](https://user-images.githubusercontent.com/1903876/77671090-54d62100-6f44-11ea-9895-fa2d0d3c7299.png)

Yeah there are two versions that come right after the other in commit order with no other changes. Maybe dependabot automatically cut a release when it fixed the bug? And then I cut one using `apm` manually?

It could also have been `apm` being buggy. I've [had issues](https://github.com/atom/apm/issues/654) with `apm` for years. Right now it doesn't actually let me publish versions the first time. It fails then I have to manually push tags and then try again. I use it so little these days that I haven't tried to figure out what's going on and fix it.

**Comment by lexicalunit** [comment]
Something happened today that made me realize what is going on finally.

![Screen Shot 2020-04-02 at 10 00 09 AM](https://user-images.githubusercontent.com/1903876/78277092-c70caf80-74c8-11ea-99e1-4b3357cf0e2f.png)

Notice how the release for 1.22.0 is from 7 days ago, yet the commits before and after it are from less than 30 minutes ago. This is because `apm publish` has been on the fritz for me for a while now and I finally figured out why.

I didn't realize that `apm` was trying to use `https` to interface with github.com. I have the repo checked out via `ssh` so I assumed `apm` would use that. Apparently it just doesn't. Every time I tried to publish a version with `apm` I'd get an error and it would seemingly only preform half of the required steps for creating a new version.

One of the final steps it was failing to do was: `git push --tags` to push up the tag and release prepare commit it was creating locally up to the remote.

This morning when I went through the process of creating a new release following another dependabot update I happened to run `git push --tags` and that is what caused the commit and tags from 7 days ago to finally be seen by github.

I've fixed the issue with `apm`, it required updating my Keychain Access credentials for github: https://help.github.com/en/github/using-git/updating-credentials-from-the-osx-keychain
